src/views/PopupView.tsx

```diff
@@ -42,6 +44,8 @@ export default function PopupView(
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
                     <dt>Body Segments</dt>
                     <dd><input type={"checkbox"} checked={showSegments} onChange={()=>setShowSegments(!showSegments)} /></dd>
+                    <dt>Export</dt>
+                    <dd onClick={exportAngles}>angles</dd>
```

Comment:
Could we make this a button to make it more obvious that it is a clickable element? Something to this effect:
```html
<dt>File Output</dt>
<dd><button onClick={exportAngles}>export angles</button></dd>
```

Reply:
It looks like this change was added?

Reply:
It hasn't yet. It's not currently a button

Reply:
not sure what to think of this but it's a button now